components/ui/Tag.jsx

```diff
@@ -0,0 +1,9 @@
+function Tag(props) {
+  return (
+    <a className={`h-fit rounded-4.75 bg-light-grey px-2 text-grey ${props.style}`} href={props.link}>
```

Comment:
This <a> anchor tag could probably be next <Link> tag

Reply:
@Delyc  By next tag do you mean "Link"???

Reply:
I understand your approach, didn't know it was to link to a different page